lib/systems/default.nix

```diff
@@ -158,38 +158,47 @@ rec {
         if final.isMacOS then "MACOSX_DEPLOYMENT_TARGET"
         else if final.isiOS then "IPHONEOS_DEPLOYMENT_TARGET"
         else null;
+    } // (
+      let
+        emulators = pkgs:
```

Comment:
Perhaps call this `selectEmulator` or something. `emulator` and `emulators` is a bit confusing.

Reply:
done